js/src/Components/RowForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {FormDialog} from "./Atoms/FormDialog";
+import {EntityForm} from "./EntityForm";
+import {Loader} from "./Atoms/Loader";
+
+export const RowForm = ({widgetContext, open, onClose, rowId, entity, loadedEntity}) => {
```

Comment:
It would be nice if this form had a message if there were no configurable options available.

Reply:
Right now the edit form opens an empty modal.

Reply:
Yeah I was wondering what to do with that. a basic message makes sense

Reply:
I added a basic message that we can improve later if necessary